modules/tinymce/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 
 ### Changed
+- Calling `tinymce.getContent({format: 'text'})` now returns newlines instead of empty string if more than one paragraph exists #TINY-8578
```

Comment:
```suggestion
- Getting the text content of the editor now returns newlines instead of an empty string if more than one empty paragraph exists #TINY-8578
```